argocd/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine as build
+ARG TARGETARCH
+WORKDIR /assets
+RUN wget https://github.com/argoproj/argo-cd/releases/latest/download/argocd-linux-$TARGETARCH -O argocd
```

Comment:
I recommend pulling down a specific version and checksumming it like [here](https://github.com/kairos-io/community-bundles/blob/main/flux/Dockerfile#L4), otherwise this could randomly get rebuilt, unintentionally upgrade and start breaking on people's installs.

Reply:
good call

Reply:
but I think dendabot will do this automagically, so let's merge I'd say

Reply:
@Itxaka do you know if this is setup for this repo too?

Reply:
it would be tracked, but only if the version is set in the dockerfile as:

```
# renovate: datasource=github-releases depName=argoproj/argo-cd
ENV VERSION=v2.12.3
```

Reply:
better ?

Reply:
BTW I need to handle both arm64 and amd64 support. For now there is only amd64